wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -135,11 +133,17 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
         backgroundHandler = Handler(backgroundThread.looper)
     }
 
-    fun init() {
+    fun initGlobal() {
         platformSyncJob = GlobalScope.launch {
-            blockchainIdentityDataDao.load()?.let {
-                blockchainIdentity = initBlockchainIdentity(it, walletApplication.wallet)
-                platformRepoInstance.initializeStateRepository()
+            init()
+        }
+    }
+
+    suspend fun init() {
+        blockchainIdentityDataDao.load()?.let {
+            blockchainIdentity = initBlockchainIdentity(it, walletApplication.wallet)
+            platformRepoInstance.initializeStateRepository()
+            platformSyncJob = GlobalScope.launch {
```

Comment:
This was changed in order to make the init() method blocking when called from other coroutine.